src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -1,13 +1,28 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Bulldozer
- * Do not remove no-args constructor
- */
 public class Bulldozer extends Machine {
+    private int weight;
+    private int height;
```

Comment:
Don't forget about access modifiers for your custom fields in `Bulldozer` class. They should be private to prevent direct access from outside of the class.